src/networking/responses/products-response.ts

```diff
@@ -1,3 +1,23 @@
+export interface Price {
+  amount: number;
+  currency: string;
+}
+
+export interface PurchaseOptionPrice {
+  period_duration: string | null;
+  price: Price | null;
+  cycle_count: number;
+}
+
+export interface PurchaseOption {
+  id: string;
+}
+
+export interface SubscriptionPurchaseOption extends PurchaseOption {
+  base_price: PurchaseOptionPrice;
+  trial: PurchaseOptionPrice | null;
+}
+
 export interface ProductResponse {
   current_price: {
```

Comment:
You could potentially substitute the type here to `Price` I think? Though we will be mostly ignoring `current_price` in future PRs I imagine and we could even remove it then.

Reply:
Absolutely, changing it!